generic-ec/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "generic-ec"
-version = "0.3.0"
+version = "0.4.0-rc1"
```

Comment:
Is my judgment right that we need a major bump if we upgrade udigest from v0.1 to v0.2? @maurges

Reply:
It's used in public APIs, so yes we should